packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/collect_multi_namespace_references.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import { isNotFoundFromUnsupportedServer } from '@kbn/core-elasticsearch-server-internal';
-import type { SavedObject } from '@kbn/core-saved-objects-common';
```

Comment:
Yeah... It kinda awkward having to do this to avoid the deprecation, given both types are supposed to be the same, but if that's the direction we're willing to take, I guess it's fine.